tests/Adapter/GeneratorTestCase.php

```diff
@@ -584,6 +587,7 @@ class Rabbits implements Animal
     const LEGS = 4;
     const SKIN = 'soft';
 
+
```

Comment:
there should be one blank line between member "types"

Reply:
That's the same issue than above.
I forgot to add tests for the 7.4 version, once done you will see that the problem exists only with the phpdoc.